datasets/ar/repet/crawler.py

```diff
@@ -86,11 +88,15 @@ def crawl_common(context: Context, data: Dict[str, str], part: str, schema: str)
     submitted_on = parse_date(data.pop("SUBMITTED_ON", None))
     listed_on = parse_date(data.pop("LISTED_ON"))
     modified_at = parse_date(data.pop("LAST_DAY_UPDATED"))
-    entity.add("createdAt", submitted_on or listed_on or modified_at)
-    entity.add("modifiedAt", modified_at)
-
-    sanction.add("listingDate", submitted_on or listed_on)
-    sanction.add("startDate", listed_on)
+    h.apply_dates(entity, "createdAt", submitted_on)
+    h.apply_dates(entity, "createdAt", listed_on)
+    if modified_at != []:
+        h.apply_date(entity, "createdAt", modified_at[0])
```

Comment:
maybe use `min(modified_at)` and then `max(modified_at)` for the next line?

Reply:
I'd suggest making both of them `modified_at`, because sometimes we have cases like: 

```
Person:createdAt,2012-02-13,
Person:createdAt,2001-02-23,
Person:modifiedAt,2012-02-13
```